modules/caddyhttp/caddyauth/basicauth.go

```diff
@@ -190,12 +192,19 @@ func (hba HTTPBasicAuth) correctPassword(account Account, plaintextPassword []by
 	if ok {
 		return same, nil
 	}
-
 	// slow track: do the expensive op, then add it to the cache
-	same, err := compare()
+    // slow track: do the expensive op, then add it to the cache
```

Comment:
Oh whoops, my bad. I left in this line by accident when I was typing the suggestion

```suggestion
```